src/main/java/com/typee/model/Tab.java

```diff
@@ -0,0 +1,28 @@
+package com.typee.model;
+
+/**
+ * Tab component for Typee Ui.
+ */
+public class Tab {
+    private String name;
+
+    public Tab(String name) {
+        this.name = name;
+    }
+
+    public Tab() {
+    }
```

Comment:
What is this constructor for?

Reply:
The default constructor was needed so that the Jackson parser can automatically parse the data into a List<Tab>. This is necessary.

Reply:
otherwise jackson will throw error..

Reply:
Ok  noted